src/nano_contracts/types.ts

```diff
@@ -73,7 +75,7 @@ export interface NanoContractHistoryAPIResponse {
 
 interface StateValueSuccess {
   // State value return
-  value: any;
+  value: NanoContractArgumentType;
```

Comment:
This can't be `Buffer` because it comes from the API

Reply:
On https://github.com/HathorNetwork/hathor-wallet-lib/pull/685/commits/ae048f821e498c422ba6450dd422a10f0cd7198c I added another type named `NanoContractArgumentApiInputType` that has no Buffer. I believe this solves this issue